internal/graph/vertex.go

```diff
@@ -24,7 +24,7 @@ func VertexID(v Vertex) interface{} {
 func VertexName(v Vertex) string {
 	switch v := v.(type) {
 	case fmt.Stringer:
-		return fmt.Sprintf("%s", v)
+		return v.String()
```

Comment:
Fixing:
```
internal/graph/vertex.go:27:10: S1025: should use String() instead of fmt.Sprintf (staticcheck)
                return fmt.Sprintf("%s", v)
```